File: TheBookLore/app/src/main/java/com/penguin/thebooklore/viewmodel/NotificationsViewModel.java
package com.penguin.thebooklore.viewmodel;

import androidx.lifecycle.LiveData;
import androidx.lifecycle.MutableLiveData;
import androidx.lifecycle.ViewModel;

public class NotificationsViewModel extends ViewModel {

    private MutableLiveData<String> mText = new MutableLiveData<>();

    public MutableLiveData<String> getText() {
        return mText;
    }

    public NotificationsViewModel() {
        mText.setValue("This is notifications fragment");
    }

}

File: TheBookLore/app/src/main/java/com/penguin/thebooklore/viewmodel/NotificationsViewModel.kt
package com.penguin.thebooklore.viewmodel

import androidx.lifecycle.LiveData
import androidx.lifecycle.MutableLiveData
import androidx.lifecycle.ViewModel

class NotificationsViewModel : BaseViewModel() {

    val text = MutableLiveData<String>()

    init {
        text.value = "This is notifications fragment"
    }

}